lib/api_me/version.rb

```diff
@@ -1,3 +1,3 @@
 module ApiMe
-  VERSION = '0.7.0'
+  VERSION = '0.7.1'
```

Comment:
This should be a major version bump for the sorting feature. I'm going to pull the bugfix out of this PR and into another PR so that I can get the fix for the perf regression in.